recipe-server/normandy/settings.py

```diff
@@ -280,7 +280,7 @@ def RAVEN_CONFIG(self):
     SECURE_PROXY_SSL_HEADER = values.TupleValue()
     SECURE_HSTS_SECONDS = values.IntegerValue(3600)
     SECURE_HSTS_INCLUDE_SUBDOMAINS = values.BooleanValue(True)
-    CSRF_COOKIE_HTTPONLY = values.BooleanValue(True)
+    CSRF_COOKIE_HTTPONLY = values.BooleanValue(False)
```

Comment:
Django still warns with a system check if this is false unless you ignore the warning, that's what's breaking the contract tests for this PR.